session_csrf/tests.py

```diff
@@ -13,11 +13,13 @@
 from django.core.handlers.wsgi import WSGIRequest
 from django.core.exceptions import ImproperlyConfigured
 
-import mock
-
 import session_csrf
 from session_csrf import (anonymous_csrf, anonymous_csrf_exempt,
                           CsrfMiddleware, prep_key)
+try:
+    import mock
+except ImportError:
+    from unittest import mock
```

Comment:
Shouldn't these be the other way around?
If you have the fancy new built-in mock from the standard library that should be used first. Only if that fails should be need the pypi library version. 
Also, a comment would be good here to remind us all why and which.